MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -688,6 +688,13 @@ private MediaStream GetMediaStream(bool isAudio, MediaStreamInfo streamInfo, Med
                 {
                     stream.BitDepth = streamInfo.BitsPerRawSample;
                 }
+
+                // mp4 missing track title workaround: fall back to handler_name if populated
+                string handlerName = GetDictionaryValue(streamInfo.Tags, "handler_name");
+                if (string.IsNullOrEmpty(stream.Title) && !string.IsNullOrEmpty(handlerName))
```

Comment:
Granted, dictionary lookups are very fast, but it's a waste if `stream.Title` isn't null or empty.

Reply:
Fair enough, re-committed with the if split so it only checks the dictionary after the title check fails.